microsoft-edge/webview2/release-notes.md

```diff
@@ -14,18 +14,33 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
 The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence.  Review the following content for up-to-date information on product announcements, additions, modifications, and breaking changes to the APIs.  
 
 > [!NOTE]
-> Ensure you re-compile your app after updating the NuGet package.  The WebView team recommends that you use the Canary channel when you develop using the prerelease packages, and the Evergreen WebView2 Runtime when you use the release packages.  For more information, navigate to [Matching WebView2 Runtime versions][Webview2ConceptsVersioningMatchingWebview2RuntimeVersions].  
+> Make sure to re-compile your app after updating the NuGet SDK package.  The WebView team recommends that you use the Canary channel of Microsoft Edge when you develop using the prerelease packages, and use the Evergreen WebView2 Runtime when you use the release packages.  For more information, navigate to [Matching the Runtime version with the SDK version][Webview2ConceptsVersioningMatchingWebview2RuntimeVersions].
 
 > [!NOTE]
-> WebView2 bug fixes are either Runtime or SDK specific.  
+> WebView2 bug fixes are either Runtime-specific or SDK-specific.  
+
+
+## Minimum version of browser and Runtime for core API support
+
+The minimum version of Microsoft Edge that supports the core APIs of the WebView2 SDK is 86.0.616.0.  This is relevant for a development version of a WebView2 app, using a non-stable Microsoft Edge release.
```

Comment:
I don't think 'support the core APIS of the WebView2 SDK' is correct. Can we not say minimum version to load WebView2?